joplin/base/management/commands/loadcontent.py

```diff
@@ -72,7 +73,7 @@ def load_process(data):
     print('✅')
 
     print(f'-  Loading process steps...\r', end='')
-    process_steps = data.pop('process_steps');
+    process_steps = data.pop('process_steps')
```

Comment:
you no like my decorative semi-colon? 😢

Reply:
I'm sorry I removed your best contribution to the code